dvc/command/data_sync.py

```diff
@@ -98,7 +98,7 @@ def add_parser(subparsers, _parent_parser):
     from dvc.command.status import CmdDataStatus
 
     # Pull
-    PULL_HELP = "Pull data files from a DVC remote storage."
+    PULL_HELP = "Download tracked files or directories from remote storage."
```

Comment:
Not sure `pull` and `fetch` help messages are different enough now. But it is definitely not worse.

Reply:
Good point. I'm just copying from https://dvc.org/doc/command-reference/fetch and https://dvc.org/doc/command-reference/pull here. This whole PR is about matching docs which I think is desirable but if not, please lmk!